Forte.EpiServer.AzureSearch/Model/DefaultDocumentBuilder.cs

```diff
@@ -68,6 +68,8 @@ public virtual T Build(IContent content)
             {
                 document.ContentTypeName = pageData.PageTypeName;
                 document.StopPublishUtc = pageData.StopPublish.HasValue ? new DateTimeOffset(pageData.StopPublish.Value) : (DateTimeOffset?) null;
+                document.CreatedAt = pageData.Created;
+                document.SortIndex = pageData.SortIndex;
```

Comment:
yes, I agree this might be misleading. My idea when adding this field was that it's better to have it and be able to use it (with proper understanding) than not have it - so (only?) use case is to get children of a parent sorted same way as EpiServer does that. 

Maybe it's an issue of proper naming this field? When renamed to ie EpiSortIndexProperty it is more explicit?